packages/manager/modules/pci/src/projects/project/data-processing/terminate-job/terminate-job.module.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+import component from './terminate-job.component';
+import routing from './terminate-job.routing';
+
+const moduleName = 'ovhManagerDataProcessingQuickTerminateModalComponent';
+
+angular
+  .module(moduleName, [])
```

Comment:
Please inject the depencies if you use them:

```suggestion
import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
import uiRouter from '@uirouter/angularjs';
import angularTranslate from 'angular-translate';
import 'ovh-ui-angular';

  angular
    .module(moduleName, [
    angularTranslate,
    ngTranslateAsyncLoader,
    'oui',
    uiRouter,
  ])
```